test/test_podman_baseline.sh

```diff
@@ -504,6 +504,16 @@ EOF
         echo "failed"
     fi
 
+    #Expected to pass (as root wit --privileged).
+    #Note that the profile should not be loaded letting the mount succeed.
+    podman run --privilged --security-opt apparmor=$aaProfile docker.io/library/alpine:latest sh -c "mkdir tmp2; mount --bind tmp tmp2"
+    rc=$?
+    echo -n "root with specified AppArmor profile but --privileged: "
+    if [ $rc == 0 ]; then
+        echo "passed"
+    else
+        echo "failed"
+    fi
```

Comment:
This test only gets run "by hand".  I like the addition, but do we need anything in the CI test suite too?

Reply:
That's a very good questions. Previously, we could not run the tests as Travis disabled AppArmor in their Ubuntu instances but maybe we can run the tests in Cirrus. @cevich, do you know if we can run AppArmor tests in the Ubuntu Cirrus instances?

Reply:
Yes, and if it's not working out of the box, we own the images (or a copy of them at least) so should be able to make it work.